content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst

```diff
@@ -25,214 +33,53 @@ barcode nomenclature options.
-+============+==========================+======+==================================+==========================================+
-| Product    | Global Trade Item Number | 01   | (01)(\\d{14})                    | :guilabel:`Barcode` field on product form|
-|            | (GTIN)                   |      |                                  |                                          |
-+------------+--------------------------+------+----------------------------------+------------------------------------------+
-| Quantity   | Variable count of items  | 30   | (30)(\\d{0,8})                   | :guilabel:`Units` field on transfer form |
-+------------+--------------------------+------+----------------------------------+------------------------------------------+
-| Lot Number | Batch or lot number      | 10   | (10)([!"%-/0-9:-?A-Z_a-z]{0,20}) | :guilabel:`Lot` on Detailed Operations   |
-|            |                          |      |                                  | pop-up                                   |
-+------------+--------------------------+------+----------------------------------+------------------------------------------+
-
-.. _barcode/operations/lot-setup:
-
-Configuration
-~~~~~~~~~~~~~
-
-To track products using lots, first enable the :ref:`Lots and Serial Numbers
-<inventory/management/track_products_by_lots>` feature. To do so, navigate to
-:menuselection:`Inventory app --> Configuration --> Settings`. Next, under the
-:guilabel:`Traceability` heading, check the box for :guilabel:`Lots & Serial Numbers`.
-
-Then, set up the product barcode by navigating to the intended product form in
-:menuselection:`Inventory app --> Products --> Products` and selecting the product. On the product
-form, click :guilabel:`Edit`. Then, in the :guilabel:`General Information` tab, fill in the
-:guilabel:`Barcode` field with the 14-digit `Global Trade Item Number (GTIN)
-<https://www.gs1.org/standards/get-barcodes>`_, which is a universal and unique identifying number
-from GS1.
-
-.. important::
-   On the product form, omit the :abbr:`A.I. (application identifier)` `01` for GTIN product barcode
-   pattern, as it is only used to encode multiple barcodes into a single barcode that contains
-   detailed information about the package contents.
+Use GS1 barcode in Odoo
+=======================
 
-.. example::
-   To create a barcode for the product, `Fuji Apple`, enter the 14-digit GTIN `12345678901231` in
-   the :guilabel:`Barcode` field on the product form.
+For product identification using GS1 barcodes in Odoo, businesses obtain a `unique GTIN
+<https://www.gs1.org/standards/get-barcodes>`_ as an internationally distinct product identifier
+purchased from GS1. This |GTIN| is combined with specific product details following GS1's designated
+*barcode pattern*. The barcode pattern's arrangement of numbers and letters must adhere to GS1
+conventions for accurate interpretation by global systems along the supply chain.
```

Comment:
@StraubCreative Do you think this makes it clear enough about GTINs that average Joe eCommerce dumdums wouldn't get confused?